File: BLH.ApproveIQ.Application/DTOs/AuditLogDto.cs
namespace BLH.ApproveIQ.Application.DTOs;

public class AuditLogDto
{
    public Guid? Id { get; set; }
    public string? Action { get; set; }
    public Guid EntityId { get; set; }
    public string? EntityName { get; set; }
    public string? OldValues { get; set; }
    public string? NewValues { get; set; }
    public string? AffectedColumns { get; set; }
    public Guid CreatedUserId { get; set; }
    public string? CreatedUserName { get; set; }
    public DateTime CreatedOnUtc { get; set; }
}

File: BLH.ApproveIQ.Portal/ClientApp/src/app/data/data/b-l-h/dtos/audit-log-dto.ts
export class AuditLogDto {
    id?: string;
    action: string;
    entityId: string;
    entityName: string;
    oldValues: string;
    newValues: string;
    affectedColumns: string;
    createdUserId: string;
    createdUserName: string;
    createdOnUtc = new Date();
}
